src/SDKs/MachineLearningCompute/MachineLearningCompute.Tests/Tests/MachineLearningComputeTests.cs

```diff
@@ -0,0 +1,198 @@
+﻿using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+using Xunit;
+using MachineLearningCompute.Tests.Helpers;
+using Microsoft.Azure.Management.MachineLearningCompute;
+using Xunit.Abstractions;
+using Microsoft.Azure.Management.MachineLearningCompute.Models;
+using System.Linq;
+
+namespace MachineLearningCompute.Tests
+{
+    public class MachineLearningComputeTests
+    {
+        private readonly ITestOutputHelper output;
+        private const string resourceGroupLocation = "East US 2";
```

Comment:
We shoudl avoid hardcoding locations - it is better to check for locations with the available resources.  This tends to create issues in maintenance when a future developer or contributor needs to re-record thsi test against a different environment

Reply:
Note: this can be fixed post-checkin if you file an issue for the next milestone, assign to  your teamand mark it as 'test debt'

Reply:
I can make the change now. Do you have a good example SDK that shows the correct pattern for handling the location?

Reply:
Yes, although you probably want to choose a default if it is in the list

Reply:
e.g.. prefer west us if there, otherwise, take the first